source/user-manual/configuring-cluster/index.rst

```diff
@@ -0,0 +1,192 @@
+
+    **Installing Splunk forwarder:**
+
+    +-------------------------------------------------------------------+-------------------------------------------------------------+
+    | Type                                                              | Description                                                 |
+    +===================================================================+=============================================================+
+    | :ref:`RPM/DEB packages <splunk_forwarder>`                        | Install Splunk forwarder for RPM or DEB based OS.           |
+    +-------------------------------------------------------------------+-------------------------------------------------------------+
+
+
+.. _deploy_wazuh_agents_cluster:
+
+.. topic:: Pointing agents to the cluster nodes
+
+    Finally, the configuration of the agents has to be modified in order to report to the cluster.
+    In this case the agent 001 will report to the worker01-node node, to achieve this it is necessary to modify the information contained in the labels ``<client><server>`` in the file ``/var/ossec/etc/ossec.conf``, inside we will place the IP of the node we want to report, in our case it would be like this:
+
+        .. code-block:: xml
+
+            <client>
+                <server>
+                    <address>WORKER01_NODE_IP</address>
+                    ...
+                </server>
+            </client>
+
+    Restart the agent:
+
+        .. code-block:: console
+
+            # systemctl restart wazuh-agent
+
+    The second agent will report to the master-node node, for this we will place the IP of our master in the agent configuration, as we have seen in the previous case:
+
+        .. code-block:: xml
+
+            <client>
+                <server>
+                    <address>MASTER_NODE_IP</address>
+                    ...
+                </server>
+            </client>
+
+    Restart the agent:
+
+        .. code-block:: console
+
+            # systemctl restart wazuh-agent
+
+
+    Let's execute the following command (works on both, worker and master nodes) to check that the agents are correctly connected:
+
+        .. code-block:: console
+
+            # /var/ossec/bin/agent_control -l
+
+            Wazuh agent_control. List of available agents:
+                ID: 000, Name: agent000 (server), IP: 127.0.0.1, Active/Local
+                ID: 001, Name: agent001, IP: 172.18.0.5, Active
+                ID: 002, Name: agent002, IP: 172.18.0.6, Active
+
+    .. important::
+
+        We recommend using a :ref:`load balancer <load_balancer>` for connecting the agents. This way the agents will be able to report to the nodes in a distributed way and it will be the balancer the one who assigns to which worker they report, this way the load will be more distributed and in case of a fall in some worker node, their agents will **reconnect** to another one.
```

Comment:
```suggestion
        We recommend using a :ref:`load balancer <load_balancer>` for connecting the agents. This way the agents will be able to report to the nodes in a distributed way and it will be the load balancer who assigns which worker they report to. Using this option we can better distribute the load and in case of a fall in some worker node, its agents will **reconnect** to another one.
```